Makefile

```diff
@@ -129,7 +134,7 @@ update:
 # setting CONTIV_NODES=<number> while calling 'make demo' can be used to bring
 # up a cluster of <number> nodes. By default <number> = 1
 start:
-	CONTIV_DOCKER_VERSION="$${CONTIV_DOCKER_VERSION:-$(DEFAULT_DOCKER_VERSION)}" CONTIV_NODE_OS=${CONTIV_NODE_OS} vagrant up
```

Comment:
Why are you making this change here? It seems unrelated to the purpose of the PR.

Reply:
was reducing duplication (the build.sh defaults this to same value), but unrelated to PR, backing this out